nss-day-cohort-47/holiday-road-salt-lake-skaters
# Em itinerary save

# Description

save button is posting saved itineraries to the json server. we still need to write functions to use that information to populate the DOM

Fixes # (issue)

## Type of change

Please delete options that are not relevant.



# Testing Instructions

run your json server and serve up the page. then select an option from each of the dropdown menu's and click save. finally go into your api/db.json and make sure that the itinerary was added to the list of objects in db.json



# Checklist:
